pom.xml

```diff
@@ -1950,10 +1998,27 @@
       </activation>
       <properties>
         <hbase.profile>2.6</hbase.profile>
+        <!-- Same compat version as 2.6.0, but different Hadoop version -->
         <hbase.compat.version>2.6.0</hbase.compat.version>
         <hbase.version>${hbase-2.6.runtime.version}</hbase.version>
       </properties>
     </profile>
+    <profile>
+      <!-- This WILL work with the public -hadoop3 artifacts -->
+      <id>phoenix-hbase-compat-2.6.0</id>
```

Comment:
Seems clever, does not require new module right?

Reply:
Yes, the HBase API is stable, earlier HBase versions just don't work with Hadoop 3.4.